code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -140,6 +140,7 @@
 	fire_sound = 'sound/magic/staff_door.ogg'
 	icon_state = "doorwand"
 	max_charges = 20 //20, 10, 10, 7
+	no_den_usage = 1
```

Comment:
Best to use the TRUE/FALSE defines for boolean values like this.

Reply:
I just went off what the other no_den_usage checks were using, but I can change it, sure.